commit 453fb2cb657c1c8bb705d68219a19f5669a4360b
Author: Henry Ptasinski <henryp@broadcom.com>
Date:   Fri Oct 1 14:05:22 2010 -0700

    staging: brcm80211: Remove unused field from wlc_info structure.
    
    Remove an unused field from the wlc_info structure, so that the underlying
    structure can also be cleaned up.
    
    Signed-off-by: Henry Ptasinski <henryp@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/sys/wlc_mac80211.h
+++ b/drivers/staging/brcm80211/sys/wlc_mac80211.h
@@ -713,7 +713,6 @@ struct wlc_info {
 	int8 ofdm_40txbw;	/* 11N, ofdm tx b/w override when in 40MHZ mode */
 	int8 mimo_40txbw;	/* 11N, mimo tx b/w override when in 40MHZ mode */
 	ht_cap_ie_t ht_cap;	/* HT CAP IE being advertised by this node */
-	ht_add_ie_t ht_add;	/* HT ADD IE being used by this node */
 
 	uint seckeys;		/* 54 key table shm address */
 	uint tkmickeys;		/* 12 TKIP MIC key table shm address */
